web/thesauruses/kotlin/1.5/io.json

```diff
@@ -0,0 +1,270 @@
+{
+    "meta": {
+        "language": "kotlin",
+        "language_name": "Kotlin",
+        "structure": "io",
+        "language_version": "1.5"
+    },
+    "concepts": {
+    "write_line": {
+        "code": [
+        "print(\"...\")"
+        ]
+    },
+    "write_line_with_new_line": {
+        "code": [
+        "println(\"...\")"
+        ]
+    },
+    "read_line": {
+        "code": [
+        "val variable_name = readln()"
+        ]
+    },
+    "read_char": {
+        "code": [
+        "val variable_name = readln()",
+        "val first_char = variable_name[0]"
+        ],
+        "comment": [
+        "First character can be read by accessing the first element of the string"
+        ]
+    },
+    "clear_console_output": {
+        "not-implemented": true
+    },
+    "change_console_background_color": {
+        "not-implemented": true
+    },
+    "change_console_text_color": {
+        "not-implemented": true
+    },
+    "file_functions_lib": {
+        "code": [
+        "java.io.File"
+        ]
+    },
+    "list_directory": {
+        "code": [
+        "val dir = File(\".\")",
+        "val files = dir.listFiles()",
+        "file?.forEach { println(it.name) }"
+        ]
```

Comment:
Similarly, as a reference tool, this should probably be just `dir.listFiles` or like `File(".").listFiles()` to show the form of how it works, not the full example.

(Similar to others below)

Reply:
Has been changed to:

![image](https://github.com/user-attachments/assets/bcab4436-a651-4ef9-80ad-719533228343)